utils/prop-types/package.json

```diff
@@ -9,16 +9,25 @@
   },
   "license": "MIT",
   "author": "Rocio Baena <rocio.baena@espressive.com>",
+  "main": "dist/index.js",
   "module": "src/index.js",
```

Comment:
We should fix the `main` and `module` paths to be the same as they are in the Cascara package as these values are used directly in the rollup config to know where to output files.

If the Rollup config was run with these values in the package.json, you may have seen the files in src getting overwritten. I would even take a look at the index.js file there specifically as the Rollup config should be looking at this and it is essentially going to try to output esm directly into the src directory with this.

Reply:
working on it 👍

Reply:
This was updated as requested